packages/data/src/onboarding/action-types.js

```diff
@@ -6,6 +6,20 @@ const TYPES = {
 	GET_PAYMENT_METHODS_SUCCESS: 'GET_PAYMENT_METHODS_SUCCESS',
 	GET_FREE_EXTENSIONS_ERROR: 'GET_FREE_EXTENSIONS_ERROR',
 	GET_FREE_EXTENSIONS_SUCCESS: 'GET_FREE_EXTENSIONS_SUCCESS',
+	GET_TASK_LISTS_ERROR: 'GET_TASK_LISTS_ERROR',
+	GET_TASK_LISTS_SUCCESS: 'GET_TASK_LISTS_SUCCESS',
+	DISMISS_TASK_ERROR: 'DISMISS_TASK_ERROR',
+	DISMISS_TASK_REQUEST: 'DISMISS_TASK_REQUEST',
+	DISMISS_TASK_SUCCESS: 'DISMISS_TASK_SUCCESS',
+	UNDO_DISMISS_TASK_ERROR: 'DISMISS_TASK_ERROR',
+	UNDO_DISMISS_TASK_REQUEST: 'DISMISS_TASK_REQUEST',
+	UNDO_DISMISS_TASK_SUCCESS: 'DISMISS_TASK_SUCCESS',
```

Comment:
The string values are missing `UNDO_`, meaning it's never hitting the intended undo logic in the reducer